src/components/Home.js

```diff
@@ -40,7 +40,7 @@ class Home extends Component {
                     <Button bsStyle='primary'>Explore Clubs</Button>
                 </Link>
                 <Link to='/create-account'>
-                    <Button bsStyle='secondary'>Create Account</Button>
+                    <Button>Create Account</Button>
```

Comment:
I'm pretty sure the error in the console about this is a bug in the library. Also removing this bsStyle changes the styling of the button. It needs to match @newlegna's design: https://www.figma.com/file/J5H98x6IAk5YVNdfOrpsZ68q/Clubs-app-sketch-file?node-id=0%3A1.

Reply:
Wait no, I'm wrong. There is no `secondary` bsStyle, I guess it should be `default` (i.e. empty like you have it. But we still need to update the SCSS to apply the right styles to this button so it still looks the same.

Reply:
?? The error was because we were passing in `secondary` to the `bsStyle` prop but the `bsStyle` prop does not accept `secondary`. 
https://react-bootstrap.github.io/components/buttons/

The button looked pretty similar to me when i removed the `bsStyle` but I didn't take a close look at it